lib/core/enums/local_settings.dart

```diff
@@ -41,6 +41,10 @@ enum LocalSettings {
   useAdvancedShareSheet(name: 'setting_use_advanced_share_sheet', label: 'Use Advanced Share Sheet'),
   showCrossPosts(name: 'setting_show_cross_posts', label: 'Show Cross-Posts'),
 
+  // Advanced Settings
+  userFormat(name: 'user_format', label: 'User Format'),
```

Comment:
If we're using `l10n` for localization, I think we can leave the `label` blank! That'll help a bit when we eventually deprecate the `label` field since it no longer applicable

Reply:
Done!